spec/factories/service_providers.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :service_provider do
     sequence(:friendly_name) { |n| "test-service_provider-#{n}" }
-    sequence(:issuer) { |n| "test-service_provider-#{n}" }
+    sequence(:issuer) { |n| "urn:gov:gsa:SAML:2.0.profiles:sp:sso:DEPT:APP-#{n}" }
```

Comment:
I would aim to only change in the UI and leave the data model the same.

- For new issuers and ones that parse correctly, we could break out two fields.
- For old issuers that do not parse correctly we could just fall back and render a single field?

Just an idea? I don't feel that strongly about it either way